modules/products/pycharm/src/main/kotlin/com/metalbear/mirrord/products/pycharm/PythonCommandLineProvider.kt

```diff
@@ -30,8 +30,13 @@ class PythonCommandLineProvider : PythonCommandLineTargetEnvironmentProvider {
 
             service.execManager.wrapper("pycharm", runParams.getEnvs()).apply {
                 this.wsl = wsl
-            }.start()?.first?.let { env ->
-                for (entry in env.entries.iterator()) {
+            }.start()?.let { executionInfo ->
+                executionInfo.envToUnset?.let { envToUnset ->
+                    for (key in envToUnset.iterator()) {
+                        executionInfo.environment.remove(key)
+                    }
```

Comment:
This will unset the variable only if it was fetched from the remote

Reply:
yes we don't have a way to remove otherwise ;(

Reply:
`pythonExecution` gives you access to raw `envs` map :+1: